tests/ci/cdk/cdk/ecr_stack.py

```diff
@@ -14,3 +14,13 @@ def __init__(self, scope: Construct, id: str, repo_name: str, **kwargs) -> None:
         repo = ecr.Repository(scope=self, id=id, repository_name=repo_name)
         repo.grant_pull_push(iam.ServicePrincipal("codebuild.amazonaws.com"))
         repo.grant_pull(iam.ArnPrincipal("arn:aws:iam::222961743098:role/scrutini-ecr"))
+        repo.add_lifecycle_rule(
+            description="Expire images older than 1 month",
+            max_image_age=Duration.days(30),
+        )
```

Comment:
Just to sanity check, let's say we have an issue that blocks our pipeline for a month, and we hit the 30 day age in production. Can we ensure we don't delete this if there is no newer image?

Reply:
Added a lifecycle policy to retain `_latest` images. Let me know if that suffice